src/test/java/ttt/PerfectPlayerTest.java

```diff
@@ -139,20 +141,20 @@ public void playsADrawWhenSecond() {
     public void winsIfFirst() {
         int cpuChoice = computerLocation();
```

Comment:
the name `cpuChoice` confused me for a moment, as I was thinking of the CPU rather than `computer`. I would use the long name to be explicit.